bundle/manifests/skupper-operator.clusterserviceversion.yaml

```diff
@@ -2,7 +2,26 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    alm-examples: '[]'
+    alm-examples: |-
+      [
+        {
+          apiVersion: v1
+          kind: ConfigMap
+          data:
+            console: "true"
+            console-authentication: internal
+            console-password: "admin"
+            console-user: "admin"
+            name: skupper-site
+            router-console: "true"
+            router-mode: interior
+            service-controller: "true"
```

Comment:
I'd suggest not including many of these options, focussing perhaps on those most likely to want editing (e.g. console-authentication related)

Reply:
The issue I have right now is that this example is not showing up at the console.
I believe only examples for CRDs introduced by the operator gets displayed.

Reply:
Shame (I wondered if that would be the case). Still it is easy enough to create a configmap through the openshift console. We can probably ensure that the doc most visible for operator users points them in that direction.